system/jlib/jlog.hpp

```diff
@@ -127,10 +113,6 @@ inline const char * LogMsgClassToVarString(LogMsgClass msgClass)
         return("Information");
     case MSGCLS_progress:
         return("Progress");
-    case MSGCLS_legacy:
```

Comment:
legacy hasn't been removed yet - here and below

Reply:
@ghalliday I'll be removing legacy in a separate ticket (https://track.hpccsystems.com/browse/HPCC-21442).  I'll mention that in this ticket.

Reply:
which is why this code should not be removed in this PR...

Reply:
oops